Factory/PsrHttpFactory.php

```diff
@@ -143,7 +143,7 @@ public function createResponse(Response $symfonyResponse)
                 $symfonyResponse->sendContent();
                 ob_end_clean();
             } else {
-                $stream->write($symfonyResponse->getContent());
+                $stream->write(is_string($symfonyResponse->getContent()) ? $symfonyResponse->getContent() : '');
```

Comment:
Maybe just this?

```suggestion
                $stream->write((string) $symfonyResponse->getContent());
```

Reply:
Yeah I thought about this but I'm not sure what`false` will be when it is converted to string.

Reply:
https://3v4l.org/Nnba7

Reply:
cool. Thank you. I've just fixed it.